main.go

```diff
@@ -147,10 +156,27 @@ var rootCmd = &cobra.Command{
 		if duration < time.Minute {
 			interval = 100 * time.Millisecond
 		}
+
+		// choose the gradient based on the flag
+		// if the flag is empty or "default", use the default gradient
+		var progressBar progress.Model
+		if gradientFlag == "" || gradientFlag == "default" {
+			progressBar = progress.New(progress.WithDefaultGradient())
+		} else if preset, ok := gradientPresets[gradientFlag]; ok {
+			progressBar = progress.New(progress.WithGradient(preset[0], preset[1]))
+		} else if strings.Contains(gradientFlag, ",") {
+			parts := strings.SplitN(gradientFlag, ",", 2)
+			progressBar = progress.New(progress.WithGradient(strings.TrimSpace(parts[0]), strings.TrimSpace(parts[1])))
+		} else {
+			// fallback to default gradient if invalid input
+			fmt.Printf("Warning: Invalid gradient '%s'. Falling back to default gradient.\n", gradientFlag)
+			progressBar = progress.New(progress.WithDefaultGradient())
+		}
```

Comment:
this can be simplified.

the default value is already "default" - so it'll never be empty

this can be something like:

```go
var progressBar progress.Model
if preset, ok := gradientPresets[gradientFlag]; ok {
  progressBar = progress.New(progress.WithGradient(preset[0], preset[1]))
} else if strings.Contains(gradientFlag, ",") {
  parts := strings.SplitN(gradientFlag, ",", 2)
  progressBar = progress.New(progress.WithGradient(strings.TrimSpace(parts[0]), strings.TrimSpace(parts[1])))
} else {
  return fmt.Errorf("invalid gradient %q", gradientFlag)
}
```